adoc/bp_chap_suma_backup.adoc

```diff
@@ -291,7 +315,7 @@ Creating a user and syncing the channels:
 * SLE-Manager-Tools12-Updates-x86_64-SP2
 
 
-Postgresql will generate an additional ~1 GB of data.
+PostgreSQL will generate an additional ~1 GB of data.
```

Comment:
Ooh, I'm not sure about using a tilde to indicate "roughly". Can we reword this?